src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
```

Comment:
```suggestion
    private static Random random = new Random();
    
    public static String getRandomColor() {
        int index = random.nextInt(Colors.values().length);
```
A good practice is to create 1 static instance of Random class, as a class field
Otherwise a new instance of this class would be created after each invocation of getRandomColor() method
And watch for empty lines, remove if unnecessary